src/main/java/hudson/plugins/cobertura/CoberturaPublisher.java

```diff
@@ -104,6 +107,27 @@ public CoberturaPublisher(String coberturaReportFile, boolean onlyStable, boolea
     }
 
     /**
+     * Legacy constructor
+     *
+     * @param coberturaReportFile
+     * @param onlyStable
+     * @param failUnhealthy
+     * @param failUnstable
+     * @param autoUpdateHealth
+     * @param autoUpdateStability
+     * @param zoomCoverageChart
+     * @param failNoReports
+     * @param sourceEncoding
+     * @param maxNumberOfBuilds
+     */
+    public CoberturaPublisher(String coberturaReportFile, boolean onlyStable, boolean failUnhealthy, boolean failUnstable,
```

Comment:
It would be better to mark this constructor as deprecated. Such Javadoc without param descriptions makes no sense. It will also affect builds with JDK8, which run javadoc lint by default